pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[tool.poetry]
+name = "rewaave"
+version = "0.1.0"
+description = "A bridge of aave staticATokens for reward collection on L2"
+authors = ["AAVE", "nethermind"]
+
+[tool.poetry.dependencies]
+python = "^3.7.2"
+cairo-lang = "^0.8.1"
```

Comment:
We don't need cairo-lang as direct dependency, starknet-devnet has cairo-lang as a dependency so we can just rely on it to get the right version

Reply:
I'd rather have it explicit than having someone change the starknet-devnet and not realise there was an update to cairo-lang